components/adblock_rust_ffi/src/wrapper.cc

```diff
@@ -44,6 +44,10 @@ Engine::Engine() : raw(engine_create("")) {}
 
 Engine::Engine(const std::string& rules) : raw(engine_create(rules.c_str())) {}
 
+Engine::Engine(const std::string& rules, const bool include_redirect_urls)
+    : raw(engine_create_with_redirect_urls(rules.c_str(),
+                                           include_redirect_urls)) {}
```

Comment:
Would it be possible to use a `ForTest` suffix on the method here (I guess it'd have to be a separate factory method)? For now we only need an engine which _parses_ redirect rules in tests, so it'd be good to have a way to keep that from sneaking into production code.

Reply:
Hm I generally like to avoid having special test code, and in this case it seemed like we might need this in the future... but I can change if you or others have strong feelings about it.